server/src/database/typeorm-seeds/metadata/field-metadata/activity-target.ts

```diff
@@ -34,6 +39,56 @@ export const seedActivityTargetFieldMetadata = async (
     ])
     .orIgnore()
     .values([
+      // Default fields
+      {
+        id: SeedActivityTargetFieldMetadataIds.Id,
+        objectMetadataId: SeedObjectMetadataIds.ActivityTarget,
+        isCustom: false,
+        workspaceId: SeedWorkspaceId,
+        isActive: true,
+        type: FieldMetadataType.UUID,
+        name: 'id',
+        label: 'Id',
+        targetColumnMap: {
+          value: 'id',
+        },
+        description: undefined,
+        icon: undefined,
+        isNullable: true,
+        // isSystem: true,
```

Comment:
Need to be there once we have added the new isSystem bool so we can hide this field.